spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -484,6 +547,58 @@ private void activateOptions() {
 			this.senderPool = Executors.newCachedThreadPool();
 		}
 
+		/**
+		 * Create the {@link ConnectionFactory}.
+		 *
+		 * @return a {@link ConnectionFactory}.
+		 */
+		protected ConnectionFactory createRabbitConnectionFactory() {
+			RabbitConnectionFactoryBean factoryBean = new RabbitConnectionFactoryBean();
+			configureRabbitConnectionFactory(factoryBean);
+			try {
+				factoryBean.afterPropertiesSet();
+				return factoryBean.getObject();
+			}
+			catch (Exception e) {
+				LogFactory.getLog("log4j2AppenderErrors").error("Failed to create customized Rabbit ConnectionFactory.",
```

Comment:
@garyrussell - Thanks.  I will make the change in both places when I make the required updates for this PR.